LINK : fatal error LNK1181: cannot open input file 'libeay32.lib'

VCPKG recently do some changes with "libeay32.lib" https://github.com/microsoft/vcpkg/pull/8566, and we got the following error when build latest version of Git, can you help look at this?

**Repro steps:**
1.open VS2017 x64 Native Tools command
2.git.exe clone -b vs/master https://github.com/git-for-windows/git D:\git\src
3.cd D:\git\src
4.git checkout "latest version"
5.devenv /upgrade git.sln
6. nuget.exe restore git.sln
7.msbuild /m /p:Configuration=Release;Platform=x64 /p:WindowsTargetPlatformVersion=10.0.17134.0 /t:Rebuild git.sln

**Failures:**
F:\git\git-http-push>"C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\VC\Tools\MSVC\14.16.27023\bin\HostX86\x64\link.exe" /ERRORREPORT:QUEUE /OUT:"..\git-http-push.exe" /NOLOGO /LIBPATH:"..\compat\vcbuild\vcpkg\installed\x64-windows\lib" advapi32.lib expat.lib kernel32.lib libcurl.lib libeay32.lib libeay32.lib libiconv.lib ntdll.lib shell32.lib ssleay32.lib ssleay32.lib user32.lib wininet.lib ws2_32.lib zlib.lib zlib.lib /MANIFEST:NO /DEBUG /PDB:"..\git-http-push.pdb" /SUBSYSTEM:CONSOLE /OPT:REF /OPT:ICF /LTCG:incremental /TLBID:1 /ENTRY:"wmainCRTStartup" /DYNAMICBASE /NXCOMPAT /IMPLIB:"..\git-http-push.lib" /MACHINE:X64 invalidcontinue.obj "x64\Release\common-main.obj" "x64\Release\http-push.obj" x64\Release\http.obj F:\gitP\KarenHuang2016\git\libgit.lib F:\gitP\KarenHuang2016\git\xdiff\xdiff_lib.lib
**LINK : fatal error LNK1181: cannot open input file 'libeay32.lib'**

--- a/compat/vcbuild/scripts/clink.pl
+++ b/compat/vcbuild/scripts/clink.pl
@@ -45,9 +45,9 @@ while (@ARGV) {
 	} elsif ("$arg" eq "-liconv") {
 		push(@args, "libiconv.lib");
 	} elsif ("$arg" eq "-lcrypto") {
-		push(@args, "libeay32.lib");
+		push(@args, "libcrypto.lib");
 	} elsif ("$arg" eq "-lssl") {
-		push(@args, "ssleay32.lib");
+		push(@args, "libssl.lib");
 	} elsif ("$arg" eq "-lcurl") {
 		my $lib = "";
 		# Newer vcpkg definitions call this libcurl_imp.lib; Do we

--- a/contrib/buildsystems/engine.pl
+++ b/contrib/buildsystems/engine.pl
@@ -343,9 +343,9 @@ sub handleLinkLine
         } elsif ("$part" eq "-lz") {
             push(@libs, "zlib.lib");
         } elsif ("$part" eq "-lcrypto") {
-            push(@libs, "libeay32.lib");
+            push(@libs, "libcrypto.lib");
         } elsif ("$part" eq "-lssl") {
-            push(@libs, "ssleay32.lib");
+            push(@libs, "libssl.lib");
         } elsif ("$part" eq "-lcurl") {
             push(@libs, "libcurl.lib");
         } elsif ("$part" eq "-lexpat") {

